content/en/docs/setup/independent/high-availability.md

```diff
@@ -306,49 +240,131 @@ the creation of additional nodes could fail due to a lack of required SANs.
         - `ETCD_1_IP`
         - `ETCD_2_IP`
 
-1.  Run `kubeadm init --config kubeadm-config.yaml` on this node.
+The following steps are exactly the same as described for stacked etcd setup:
+
+1.  Run `sudo kubeadm init --config kubeadm-config.yaml --experimental-upload-certs` on this node.
 
-1.  Write the join command that is returned to a text file for later use.
+1.  Write the output join commands that are returned to a text file for later use.
 
-1.  Apply the Weave CNI plugin:
+1.  Apply the CNI plugin of your choice. The given example is for Weave Net:
 
     ```sh
     kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
     ```
 
 ### Steps for the rest of the control plane nodes
 
-To add the rest of the control plane nodes, follow [these instructions](#steps-for-the-rest-of-the-control-plane-nodes).
-The steps are the same as for the stacked etcd setup, with the exception that a local
-etcd member is not created.
-
-To summarize:
+The steps are the same as for the stacked etcd setup:
 
 - Make sure the first control plane node is fully initialized.
-- Copy certificates between the first control plane node and the other control plane nodes.
-- Join each control plane node with the join command you saved to a text file, plus add the `--experimental-control-plane` flag.
+- Join each control plane node with the join command you saved to a text file. It's recommended
+to join the control plane nodes one at a time.
+- Don't forget that the decryption key from `--certificate-key` expires after two hours, by default.
 
 ## Common tasks after bootstrapping control plane
 
-### Install a pod network
+### Install workers
 
-[Follow these instructions](/docs/setup/independent/create-cluster-kubeadm/#pod-network) to install
-the pod network. Make sure this corresponds to whichever pod CIDR you provided
-in the master configuration file.
+Worker nodes can be joined to the cluster with the command you stored previously
+as the output from the `kubeadm init` command:
 
-### Install workers
+```sh
+sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866
+```
+
+## Manual certificate distribution {#manual-certs}
+
+If you choose to not use `kubeadm init` with the `--experimental-upload-certs` flag this means that
+you are going to have to manually copy the certificates from the primary control plane node to the
+joining control plane nodes.
+
+There are many ways to do this. In the following example we are using `ssh` and `scp`:
+
+SSH is required if you want to control all nodes from a single machine.
 
-Each worker node can now be joined to the cluster with the command returned from any of the
-`kubeadm init` commands. The flag `--experimental-control-plane` should not be added to worker nodes.
+1.  Enable ssh-agent on your main device that has access to all other nodes in
```

Comment:
Would it be simpler to document using `scp -3` and skip using an SSH agent?

(`scp -3` runs all transfers via the local client).

Reply:
in the case of `scp -3` routing we do avoid the need for ssh-agent.
but the idea of enabling ssh-agent is to also allow to ssh to the nodes and execute commands on them.